src/clustering/master/Master.ts

```diff
@@ -47,18 +47,15 @@ export class Master extends Emitter<{
 }> {
   /**
    * Options
-   * @type {BotOptions}
    */
   public options: CompleteBotOptions
   /**
    * Rest Manager (only set after running .start())
-   * @type {RestManager}
    */
-  public rest = {} as RestManager
+  public rest: RestManager = {} as RestManager
```

Comment:
You dont need to do this when removing type jsdoc.
```suggestion
  public rest = {} as RestManager
```